paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PaymentSheetScreen.kt

```diff
@@ -151,37 +151,40 @@ internal fun Wallet(
     val padding = dimensionResource(R.dimen.stripe_paymentsheet_outer_spacing_horizontal)
 
     if (containerState.shouldShow) {
-        Column(modifier = modifier.padding(horizontal = padding)) {
-            if (containerState.showGooglePay) {
-                GooglePayButton(
-                    state = googlePayButtonState?.convert(),
-                    isEnabled = buttonsEnabled,
-                    onPressed = viewModel::checkoutWithGooglePay,
-                    modifier = Modifier.padding(top = 7.dp),
-                )
+        Column {
+            Column(modifier = modifier.padding(horizontal = 4.dp)) {
+                if (containerState.showGooglePay) {
+                    GooglePayButton(
+                        state = googlePayButtonState?.convert(),
+                        allowCreditCards = viewModel.googlePayAllowCreditCards,
+                        billingAddressParameters = viewModel.googlePayBillingAddressParameters,
```

Comment:
Can we expose these as part of `ContainerState` instead? Maybe a data class nested under `ContainerState`.